db_stress_tool/db_stress_test_base.cc

```diff
@@ -129,7 +129,8 @@ std::shared_ptr<Cache> StressTest::NewCache(size_t capacity,
     }
     return cache;
   } else if (FLAGS_cache_type == "fast_lru_cache") {
-    return NewFastLRUCache((size_t)capacity, num_shard_bits);
+    return NewFastLRUCache((size_t)capacity, FLAGS_block_size, num_shard_bits,
```

Comment:
Not originally your issue, but C-style casts are bad style, potentially hazardous if code is changed. https://google.github.io/styleguide/cppguide.html#Casting  Here it seems the casts are completely unnecessary. ;)

Reply:
Understood. I'll fix this occurrence only so we don't deviate from the PR's goal.